templates/plugin/salt_master.conf.erb

```diff
@@ -0,0 +1,56 @@
+# /etc/salt/master.d/foreman.config Master configuration
+#
+# This file summarizes configurations for the salt-master.
+# Have a look at the [Foreman Salt Plugin Documentation](https://theforeman.org/plugins/foreman_salt/) for detailed explanations.
+#
+# After editing this file, run the following command to active the changes:
+# $ systemctl restart salt-master
```

Comment:
Technically the convention is that `#` examples run as root while `$` runs as non-root. However, this also looks a bit odd:
```suggestion
# # systemctl restart salt-master
```

Perhaps just simplify it to:
```suggestion
# systemctl restart salt-master
```

Users can probably guess that it needs to run as root.